performance/src/performance_dags/elastic_dag/elastic_dag.py

```diff
@@ -0,0 +1,322 @@
+"""
+Elastic dag copied from apache/airflow master and adjusted to work on python2 and with Airflow
```

Comment:
```suggestion
Elastic dag copied from apache/airflow main and adjusted to work on python2 and with Airflow
```

Reply:
Also -  I think that comment is outdated in general.

Reply:
Agree. I have updated the comment adding more information about the DAG usage.